src/mito2/src/schedule/scheduler.rs

```diff
@@ -87,7 +87,7 @@ impl LocalScheduler {
                 if state_clone.load(Ordering::Relaxed) == STATE_AWAIT_TERMINATION {
                     // recv_async waits until all sender's been dropped.
                     while let Ok(req) = receiver.recv_async().await {
-                        req.await;
+                        let _ = req.await;
```

Comment:
1. Using `job` for elements in the scheduler is more appropriate. Please also check other local variables.
2. If we always ignore job results, could we change the return value of a `Job` to `()` instead of `Result<()>`?
```suggestion
                        // Ignore job result
                        let _ = job.await;
```

Reply:
1. OK, done
2. sry, `let _ = job.await;` shoud be  `job.await;`. Initially, we intended to use `Result<()>` as the return value, but we found that even if there is an Error, we have to ignore it, that is, in the form of `let _`, so we finally consider using () as the return value